### ISSUE TITLE
Change in NYT10 dataset?

### ISSUE THREAD

**Issue by fenchri** [thunlp/OpenNRE#322]
Hi,

It appears you changed the version of the NYT10 dataset again? Previously there were 53 relations, now I find there are 58.
Also, the training instances are now 522611 where previously they were 522043?

Thanks in advance

**Comment by gaotianyu1350** [comment]
HI,

Sorry for the confusion. Previously we provide a version that has a validation set. This is not consistent from any publication, so after someone raised the issue, we changed it to the original version (Riedel et al., 2010) and will stick to this one.

**Comment by fenchri** [comment]
I see, thanks for your reply!

As far as I remember though, the previous version (a few months ago, maybe September?) also didn't have a validation set and there were 522043 training instances with 53 relations. 
If I am not mistaken most of the previous work uses the 53 relations version. 
Maybe now this is up to each one to choose the version he will use?

**Comment by gaotianyu1350** [comment]
There are 53 valid relations in the datasets. In the current (and the original) version, there are some relations that do not show up in the training set or the test set. Since the current version is the most faithful one to the original data, we decide to put this on and let people decide what modification they want to make (e.g., delete those invalid relations, or split a validation set).

**Comment by fenchri** [comment]
I understand the setting of the original data and it makes 100% sense to me to keep the original data. 
It is just that there are too many versions of the dataset out there, so it is useful to have just one for fair comparisons!

Thanks again for your reply!
